x-pack/plugins/apm/ftr_e2e/cypress/support/types.d.ts

```diff
@@ -8,13 +8,14 @@
 declare namespace Cypress {
   interface Chainable {
     loginAsViewerUser(): void;
-    loginAsEditorUser(): void;
+    loginAsEditorUser(): Promise<Cypress.Response<any>>;
```

Comment:
I expect also other `loginAs*` function to return a promise. Does it make sense to update them?